### ISSUE TITLE
'NoneType' object is not callable

### ISSUE THREAD

**Issue by Dog** [DamnWidget/anaconda#262]
I've been trying to verify that pylint is actually working. I updated to Package Control 3.0 and looked in the console to see if I was getting any errors. I get this error eventually in ST3 on Windows 7 (shortly after ST3 launches and anaconda is loaded)

```
Registering handler with address ('localhost', 55737)
ERROR:root:'NoneType' object is not callable
ERROR:root:Traceback (most recent call last):
ERROR:root:  File "C:\Users\Dog\AppData\Roaming\Sublime Text 3\Packages\Anaconda\anaconda_lib\jsonclient.py", line 94, in process_message
ERROR:root:    callback(data)
ERROR:root:  File "C:\Users\Dog\AppData\Roaming\Sublime Text 3\Packages\Anaconda\anaconda_lib\callback.py", line 73, in __call__
ERROR:root:    return self._fire_callback(*args, **kwargs)
ERROR:root:  File "C:\Users\Dog\AppData\Roaming\Sublime Text 3\Packages\Anaconda\anaconda_lib\callback.py", line 219, in _fire_callback
ERROR:root:    return callback(*args, **kwargs)
ERROR:root:TypeError: 'NoneType' object is not callable
ERROR:root:'NoneType' object is not callable
```

and this error repeats periodically

Here are my Anaconda user settings:

```javascript
{
	"use_pylint": true,
	"pep8": true,
	"pylint_rcfile": "C:\\Users\\Dog\\code\\project\\web\\core\\pylintrc",
    "pep257": false,
    "anaconda_linter_delay": 0.2,
}
```

And I have C:\Python27\Scripts and C:\Python27\ in the Path System Variable.

**Comment by DamnWidget** [comment]
I'm not sure what can be the cause for this. Seems like the registered callback to process the data that is returned from the JsonServer is not valid, two things:

* Is autocompletion, show documentation, goto and find usages working?
* Can you provide with logs from JsonServer? (They are in %APPDATA%/Jedi/Jedi/<project_name>/anaconda_jsonserver.log

**Comment by Dog** [comment]
I took a look in at the `anaconda_jsonserver.log` locationed at C:\Users\Dog\AppData\Roaming\Jedi\Jedi\core (in case someone else runs into this problem)

I found the error "Your configured python interpreter can't import pylint" several times throughout the log.

I reinstalled pylint and it solved the issue.

```
pip uninstall pylint
pip install pylint
```

FWIW, this seems to have prevented Anaconda from updating. Immediately after reinstalling pylint Package Control gave me a message Anaconda was updated to 1.3.6. I didn't realize I was on an older version.

Thanks
